tests/table/test_eval.py

```diff
@@ -491,3 +491,43 @@ def test_table_numeric_sqrt() -> None:
     assert [math.sqrt(v) if v is not None else v for v in table.get_column("b").to_pylist()] == sqrt_table.get_column(
         "b"
     ).to_pylist()
+
+
```

Comment:
Could you add tests for:
- columns with nulls
- shift left/right with a scalar value
- trying to shift with a negative number

Reply:
Done